libdnf/repo/Repo.hpp

```diff
@@ -229,6 +229,19 @@ struct Repo {
     void setMaxMirrorTries(int maxMirrorTries);
     int getTimestamp() const;
     int getMaxTimestamp();
+
+    /**
+     * @brief Try to preserve remote side timestamps
+     *
+     * When set to 1 the underlying librepo is asked to make an attempt to set the timestamps
+     * of the local downloaded files (repository metadata and packages) to match those from
+     * the remote files.
+     * This feature is by default switched off.
+     *
+     * @param preserveRemoteTime 1 - use remote file timestamp, 0 - use the current time
+    */
+    void setPreserveRemoteTime(int preserveRemoteTime);
```

Comment:
Why `int` type?  The method is proposed for enable/disable something. It is an ideal case for using `bool` type.

Reply:
Changed to bool.